prow/config.yaml

```diff
@@ -553,7 +553,7 @@ presubmits:
           value: /etc/ssh-key-secret/ssh-private
         - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
           value: /etc/ssh-key-secret/ssh-public
-        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170718-c3574ec6
+        image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170720-69a2043d
```

Comment:
Since both of them are using https://github.com/kubernetes/test-infra/pull/3611/commits/69a2043d46ec84208edb2432ea7fc88ce10328de